Tasks/AzureResourceGroupDeployment/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 2,
-        "Patch": 11
+        "Patch": 12
     },
```

Comment:
can you update the version of all the tasks consuming this package. And update the version of arm-rest..

Reply:
Done